app/src/main/java/se/sigmaconnectivity/blescanner/service/FcmService.kt

```diff
@@ -2,17 +2,16 @@ package se.sigmaconnectivity.blescanner.service
 
 import com.google.firebase.messaging.FirebaseMessagingService
 import com.google.firebase.messaging.RemoteMessage
-import io.reactivex.disposables.CompositeDisposable
+import org.json.JSONObject
```

Comment:
Paweł używał gsona do tego. nie wiem czy nie bedzie lepiej bedzie sie go trzymac to serializacji/deserializacji